seednode/torrc

```diff
@@ -1,8 +1,18 @@
-SOCKSPort 9050 # Default: Bind to localhost:9050 for local connections.
-Log notice syslog
 RunAsDaemon 1
+SOCKSPort 9050
 ControlPort 9051
+Log notice syslog
+
 CookieAuthentication 1
 CookieAuthFileGroupReadable 1
 DataDirectoryGroupReadable 1
-KeepalivePeriod 42
+
+SafeSocks 0
+HiddenServiceStatistics 0
+AvoidDiskWrites 1
+
+MaxClientCircuitsPending 64
+KeepalivePeriod 2
+#CircuitBuildTimeout 5
+#NewCircuitPeriod 15
+#NumEntryGuards 8
```

Comment:
@wiz  As I saw some messages in the explorer channel regarding tor settings. Is this torrc configuration still final?

Reply:
Okay yeah I just pushed a commit which comments out the advanced settings for now, I will address this in a separate PR